src/deepsparse/yolo/annotate.py

```diff
@@ -111,6 +113,15 @@
     help="Inference engine backend to run on. Choices are 'deepsparse', "
     "'onnxruntime', and 'torch'. Default is 'deepsparse'",
 )
+@click.option(
+    "--model_input_shape",
+    "--model-input-shape",
```

Comment:
model input shape is a bit misleading here because the model input shape would be the tensor shape (ie (1,3,640,640)) but we're looking for an input shape

Reply:
Updated name to `--model_input_image_shape`